cmd/packer-sdc/internal/renderdocs/test-data/docs/builder-docs.mdx

```diff
@@ -0,0 +1,11 @@
+Hello and welcome to the awesome docs
+
+@include 'partial-foo.mdx'
+
+Bar:
+
+@include 'bar/partial-bar.mdx'
+
+@include 'packer-plugin-sdk/communicator/Config.mdx'
```

Comment:
Nice! I think this solves one of the tasks I added to the vision board to think about these communicator shared docs.

Reply:
I tried this in the alicloud plugin extraction and it doesn't seem to be working -- 

markdown file: 

```
@include 'packer-plugin-sdk/communicator/SSH-Key-Pair-Name-not-required.mdx'
```

output from generate:
```
2021/04/14 11:41:54 renderDocsFile: open docs-partials/packer-plugin-sdk/communicator/SSH-Key-Pair-Name-not-required.mdx: no such file or directory
```

Reply:
ah, looks like somehow the referenced partial isn't in the sdk's website/content/partials/communicator. I'll investigate

Reply:
Just got the same thing here.

Reply:
Moss added it, thanks Moss !!!!